index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <script src="https://kit.fontawesome.com/b08094b2a2.js" crossorigin="anonymous"></script>
+    <title>Personal Portfolio</title>
+</head>
+
+<body>
+    <header>
+        <nav><i class="fas fa-bars"></i></nav>
+
+        <h1 class="header">Hey there.
+            I’m Ian.
+           <font class="text_color">I’m a software developer</font>
+        </h1>
```

Comment:
Great work on your project so far! I noticed some small things that could be improved to align your project with the Figma template and best practices:
- [ ] Could you please move all the h1 text inside your section, as requested by a previous reviewer, and for better `HTML` structure and semantics?
- [ ] As requested by a previous reviewer it is recommended to use an `hx` tag for your subtitle instead of a `font` tag, for better accessibility and semantics.
- [ ] The color of your subtitle should match the [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=48-988&t=mnMnPg6Wment6Asj-0) color. You can find the correct color by inspecting the element inside Figma.
- [ ]Please check the `font-size` of your text to ensure it aligns with the [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=48-988&t=mnMnPg6Wment6Asj-0). Here are the suggested values for your h1 and subtitle:
![image](https://user-images.githubusercontent.com/107952724/235245962-287eb8bd-8fdf-4dcc-9018-be397a5778a1.png)

Thank you for considering these suggestions, and keep up the great work!